src/pfe/file-watcher/scripts/kubeScripts/modify-helm-chart.sh

```diff
@@ -29,6 +29,14 @@ function addOwnerReference() {
     yq w -i $filename -- metadata.ownerReferences[$index].uid $PFE_UID
 }
 
+function overwriteContainerArgsWithTail() {
```

Comment:
so why do we want to `tail -f /app/package.json` and why are we doing this for all the proj types ie projects calling this script?

Reply:
Good spot. We want to do this for Remote Node projects so we can restart the Node process without the container dying. I've moved this function to a Node-specific file that is only called by `nodejs-container.sh`.

Does that resolve your comment?

Reply:
I would also just over ride those information directly using the helm command. If you are just modifying the charts for one property, you can just directly specify it in the helm command, rather than creating a new file for it? Can you see if this can be done..

Reply:
I looked at overriding this information directly using the helm command, but did not see how to. I tried several things, including `--set`. But that seems to override the `Values.yaml`, whereas what we need is to actually extend the helm chart with an `args` field. (The values I `--set` don't seem to be picked up unless the helm chart contains a field looking for it). Ref: https://helm.sh/docs/helm/helm_install/

If you know otherwise how to override information using just the helm command, could you link me some docs please?